docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -4,127 +4,111 @@ description: Provides instructions for upgrading Teleport Cloud agents that run
+- [Manual updates](#manual-updates)
 
 ## Prerequisites
 
 - Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
   which describes the sequence in which to upgrade components of your cluster.
 
-- Teleport Enterprise Cloud. You can determine the current version of these
-  services by running the following command, where `mytenant` is the name of
-  your Teleport Enterprise Cloud tenant. This requires the
-  [`jq` CLI tool](https://jqlang.github.io/jq/):
-
-  ```code
-  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
-  ```
-
 - (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-Teleport supports automatic agent updates for the `teleport-kube-agent` Helm
-chart. The [Automatic Update Architecture
-](../architecture/agent-update-management.mdx) guide describes how agent
-updating works. Automatic agent upgrades require:
+## Check for agent updates
+
+The latest compatible Teleport agent version is served by the Teleport Proxy. To
+identify what that version is, you can query the `webapi` endpoint.
+```code
+$ curl https://<Var name="teleport.example.com" />/webapi/automaticupgrades/channel/stable/cloud/version
+v15.2.1
+```
+
+To list the agents that are upgradable, use the `tctl inventory ls` command to list
+the inventory of connected agents. The agents are listed along with their current
+version. Additional flags are available to filter by service `--services=node,kube,db,...`,
+or to filter by version `--older-than=v15.2.1`.
+```code
+$ tctl inventory ls --older-than=v15.2.1
+Server ID                            Hostname       Services       Version Upgrader
+------------------------------------ -------------- -------------- ------- --------
+00000000-0000-0000-0000-000000000000 teleport-agent Kube           v14.4.5 none
+00000000-0000-0000-0000-000000000001 teleport-proxy Proxy          v15.2.0 none
+00000000-0000-0000-0000-000000000002 teleport-auth  Auth,Discovery v15.2.0 none
+...
+```
+Note that the `inventory ls` command will also list `teleport-auth` and `teleport-proxy`
+services. These services are managed by the Teleport Cloud team and updates will
+be performed automatically.
 
-- A Teleport Cloud account that supports automatic updates. To determine if
-  your account supports automatic updates, run the following command, replacing
-  `mytenant.teleport.sh` with the address of your Teleport Cloud account:
+## Automatic updates
 
-  ```code
-  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.automatic_upgrades'
-  true
-  ```
-- At least one Teleport Enterprise agent deployed via the `teleport-kube-agent`
-  Helm chart.
+Agents enrolled in automatic updates will automatically be kept up to date with the
+latest compatible version of Teleport. You can find more information regarding the
+automatic updates architecture in the [Agent Update Management](../architecture/agent-update-management.mdx)
```

Comment:
We should move this link to the end of the page per the [style guide](https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#using-links-admonishments-and-details-in-how-to-guides).